.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+  - repo: https://github.com/sphinx-contrib/sphinx-lint
```

Comment:
Let's put sphinx-lint last, it's slower to run so I prefer to get faster feedback from the others first.